main/tests.py

```diff
@@ -1,22 +1,23 @@
 import pytest
+
+from daycare.models import DayCare
 from dogowner.models import DogOwner
 
 
 @pytest.fixture
 def create_dog_owner_user():
-    return DogOwner.create(email='dogowner@address.com',
-                           username='dogOwnerUser01',
-                           password='password123',
-                           dog_name='dog name',
-                           first_name='test',
-                           last_name='user',
-                           phone_number=1234567890,
-                           dog_race='dog race',
-                           dog_picture_url='https://www.google.com/user1.jpg',
-                           dog_age=4,
-                           dog_weight=2,
-                           dog_gender='M'
-                           )
+    return DogOwner.create(email='dogowner@address.com', username='dogOwnerUser01', password='password123',
```

Comment:
How is this change relevant to this PR?

Reply:
It's just shrinks the place this fixture takes. I can revert it it you think it's redundant.

Reply:
Update - after the conftest file, this is no longer relevant. Deleted.